subprojects/dependency-management/src/main/java/org/gradle/api/internal/artifacts/repositories/AbstractAuthenticationSupportedRepository.java

```diff
@@ -36,71 +36,71 @@
 import java.util.List;
 
 public abstract class AbstractAuthenticationSupportedRepository extends AbstractResolutionAwareArtifactRepository implements AuthenticationSupportedInternal {
-    private final AuthenticationSupporter delegate;
+    private final AuthenticationSupporter authenticationSupporter;
```

Comment:
Why the rename?
I find that `delegate` explains a lot about how this class works

Reply:
My thinking was that when you see "delegate" in code, it's just a generic term that doesn't mean anything, whereas "authenticationSupporter" actually tells you what it is. I can revert it if you feel strongly about it, I'm fine with both versions.

Reply:
Reverted to "delegate".